sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/healthcare/HealthcareApiClient.java

```diff
@@ -248,4 +248,8 @@ FhirStore createFhirStore(String dataset, String name, String version, String pu
   Empty deleteDicomStore(String name) throws IOException;
 
   Empty uploadToDicomStore(String webPath, String filePath) throws IOException, URISyntaxException;
+
+  Operation deidentifyDicomStore(
```

Comment:
Though existing interface methods are sparsely documented - which is not good - could you please add some comment to this newly added method?

Reply:
agreed, filed https://github.com/apache/beam/issues/28415 to extend this